crates/re_components/src/load_file.rs

```diff
@@ -46,7 +46,9 @@ pub fn data_cell_from_file_path(file_path: &std::path::Path) -> Result<DataCell,
         #[cfg(feature = "image")]
         _ => {
             // Assume an image (there are so many image extensions):
-            let tensor = crate::Tensor::from_image_file(file_path)?;
```

Comment:
Looks like this needs to add the indicator components.

Reply:
We can't really add an indicator component here until that is its own code-genned struct, which I believe @teh-cmc is working on